tests/functions/test_update_where.py

```diff
@@ -1,5 +1,8 @@
 import pandas as pd
 import pytest
+from pandas._testing import assert_frame_equal
```

Comment:
Hi @samukweku can you import from `pandas.testing` instead of `pandas._testing`? I think it would be better to use the public module instead of the private one

Reply:
Not a problem. I'll make the change.